script/seed.js

```diff
@@ -26,22 +26,24 @@ async function seed() {
 
   const products = await Promise.all([
     Product.create({
-      name: 'Sloth Bed'
+      name: 'Sloth Bed',
+      imageUrl: '../public/img/bed.jpg'
```

Comment:
imageUrl needs to be from the root of the website, i.e. '/img/bed.jpg' not '../public/img/bed.jpg'